quickwit-metastore/src/split_metadata_version.rs

```diff
@@ -64,10 +65,25 @@ struct SplitMetadataV0 {
     pub tags: BTreeSet<String>,
 
     /// Number of demux operations this split has undergone.
-    #[serde(default)]
-    pub demux_num_ops: usize,
+    #[serde(default, rename = "demux_num_ops", skip_serializing)]
```

Comment:
I think we can just remove that here. serde will ignore unknown field by default.